WebAPI/appsettings.json

```diff
@@ -12,6 +12,9 @@
       }
     }
   },
+  "PathSettings": {
+    "CoursesJsonPath": "..\\CoursesParser\\ActualCoursesJson\\courses_2018_11_20.json"
```

Comment:
do we really need this in settings file? Path is wrong. You should copy the file to the output directory and load it from there.

Reply:
Problem is that there are two different project = two different outputs, if I would like to copy file from one project output to another command I need to use post build event and command for copy will be depend on operating system